### PR TITLE
Gtk3 sample gtk demo scale rb

### PR DESCRIPTION
**Pull request by cedlemo**
See issue #608

### CODE REVIEW

**Comment by cedlemo** [comment]
Rebase done 7-nov-2015

File: gtk3/sample/gtk-demo/scale.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+=end
+
+module ScaleDemo
+  def self.run_demo(main_window)
+    builder = Gtk::Builder.new(:resource => "/scale/scale.ui")
+    builder.connect_signals {}
+    window = builder["window1"]
+    window.screen = main_window.screen
+
+    window.signal_connect("destroy") { window.destroyed(window) }
```
Comment (kou) [inline gtk3/sample/gtk-demo/scale.rb]: You can just remove this line. Because it's only for C. It's not needed for Ruby.

**Comment by cedlemo** [comment]
Line removed.

**Comment by kou** [comment]
Thanks!

**Pull request merged**